libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -364,6 +365,24 @@ int32_t AP_Landing_Deepstall::get_target_airspeed_cm(void) const
     }
 }
 
+bool AP_Landing_Deepstall::send_deepstall_message(mavlink_channel_t chan) const
+{
+//    CHECK_PAYLOAD_SIZE(DEEPSTALL);
```

Comment:
@WickedShell Grab a post-it and stick it in your screen so you don't forget again :stuck_out_tongue_winking_eye:  You can use either [HAVE_PAYLOAD_SPACE(chan, id)](https://github.com/ArduPilot/ardupilot/blob/master/libraries/GCS_MAVLink/GCS.h#L21) or [CHECK_PAYLOAD_SIZE2(id)](https://github.com/ArduPilot/ardupilot/blob/master/libraries/GCS_MAVLink/GCS.h#L23) (with this last one your channel variable needs to be called chan, which it is).